packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -221,15 +224,15 @@ export function generateGrid(
         const name = String(column.name);
 
         let renderCell: string | undefined = undefined;
-        let valueGetter: string | undefined = undefined;
+        let valueGetter: string | undefined = name.includes(".") ? `({ row }) => row.${name.replace(/\./g, "?.")}` : undefined;
```

Comment:
In handmade code I wouldn't add the optional chaining parameter to required fields -> Shouldn't the generator do the same?

Reply:
you're right. but for now it's not so easy to find out which level is optional. this is a tradeoff instead of a lot more lines of code.